packages/vm/lib/evm/types.ts

```diff
@@ -0,0 +1 @@
+export type Log = [Buffer, Buffer[], Buffer] // [address, topics, data]
```

Comment:
Ok just looking one last time, I think a proper doc here should be better for editor hinting 🙂 then I can give a final approval and we can merge this in!

```suggestion
/**
 * Logs that the contract emitted.
 * Format: [address, topics, data]
 */
export type Log = [Buffer, Buffer[], Buffer]
```